### PR TITLE
RELATED: CAL-900 add explicit WDF support

### PR DESCRIPTION
**Pull request by hkad98**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-sdk/gooddata_sdk/catalog/workspace/declarative_model/workspace/logical_model/ldm.py [hunk 1]
```diff
@@ -44,54 +48,100 @@ def load_from_disk(cls, workspace_folder: Path) -> CatalogDeclarativeModel:
 class CatalogDeclarativeLdm(Base):
     datasets: List[CatalogDeclarativeDataset] = attr.field(factory=list)
     date_instances: List[CatalogDeclarativeDateDataset] = attr.field(factory=list)
+    dataset_extensions: Optional[List[CatalogDeclarativeDatasetExtension]] = None
+
+    """
+    TODO: add support for dataset_extensions load and store
```
Comment (jaceksan) [inline gooddata-sdk/gooddata_sdk/catalog/workspace/declarative_model/workspace/logical_model/ldm.py:54]: Intentional TODO, or forgotten implementation?

**Review by jaceksan — CHANGES_REQUESTED** [review]

File: gooddata-sdk/gooddata_sdk/catalog/workspace/declarative_model/workspace/logical_model/ldm.py (hunk 1 above)
Comment (hkad98) [reply gooddata-sdk/gooddata_sdk/catalog/workspace/declarative_model/workspace/logical_model/ldm.py]: Forgotten. Thanks for the catch.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by jaceksan — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR315

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Pull request merged**